.github/workflows/swift-toolchain.yml

```diff
@@ -700,6 +698,123 @@ jobs:
             ${{ steps.export-binary-paths.outputs.swift_serialize_diagnostics }}
             ${{ steps.export-binary-paths.outputs.swift_compatibility_symbols }}
 
+  early_swift_driver:
+    # TODO: Build this on Windows too.
+    if: inputs.build_os == 'Darwin'
+    needs: [sqlite]
+    runs-on: ${{ inputs.default_build_runner }}
+
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJSON(inputs.host_matrix) }}
+
+    name: ${{ matrix.os }} ${{ matrix.arch }} Early Swift Driver
+
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: sqlite-${{ matrix.os }}-${{ matrix.arch }}-${{ inputs.swift_toolchain_sqlite_version }}
+          path: ${{ github.workspace }}/BuildRoot/Library/sqlite-${{ inputs.swift_toolchain_sqlite_version }}/usr
+
+      - uses: actions/checkout@v4
+        with:
+          repository: swiftlang/swift-llbuild
+          ref: ${{ inputs.swift_llbuild_revision }}
+          path: ${{ github.workspace }}/SourceCache/swift-llbuild
+          show-progress: false
+      - uses: actions/checkout@v4
+        with:
+          repository: swiftlang/swift-tools-support-core
+          ref: ${{ inputs.swift_tools_support_core_revision }}
+          path: ${{ github.workspace }}/SourceCache/swift-tools-support-core
+          show-progress: false
+      - uses: actions/checkout@v4
+        with:
+          repository: apple/swift-argument-parser
+          ref: ${{ inputs.swift_argument_parser_revision }}
+          path: ${{ github.workspace }}/SourceCache/swift-argument-parser
+          show-progress: false
+      - uses: actions/checkout@v4
+        with:
+          repository: jpsim/Yams
+          ref: ${{ inputs.yams_revision }}
+          path: ${{ github.workspace }}/SourceCache/Yams
+          show-progress: false
+      - uses: actions/checkout@v4
+        with:
+          repository: swiftlang/swift-driver
+          ref: ${{ inputs.swift_driver_revision }}
+          path: ${{ github.workspace }}/SourceCache/swift-driver
+          show-progress: false
+
+      - uses: compnerd/gha-setup-vsdevenv@main
+        with:
+          host_arch: ${{ inputs.build_arch }}
+          components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
+          arch: ${{ matrix.arch }}
+
+      - uses: seanmiddleditch/gha-setup-ninja@master
+        if: inputs.build_os == 'Darwin'
+
+      - name: Install Swift Toolchain (Mac)
+        if: inputs.build_os == 'Darwin'
```

Comment:
On Windows, where do we install swift?

Reply:
This job does not have a Windows version yet. @compnerd is working on bringing the early swift-driver build separately.